### DISCUSSION TITLE
How to disable widget resizing

### DISCUSSION THREAD

**Discussion by Rupesh-91** [LawnchairLauncher/lawnchair#3292] [Q&A] [answered]
Is it possible to disable widget resizing. Currently i have tried with resizeMode: None but its not working.

**Comment by SuperDragonXD** [comment, marked as answer]
Hello. I know that this is a bit late, but you can disable it by setting that manifest value in your widget *and* disabling the "Enforce widget resizing" and "Remove size constraints" options in Home settings > Home screen.
